app/models/package_manager/pypi.rb

```diff
@@ -71,5 +71,11 @@ def self.dependencies(name, version, _project)
         }
       end
     end
+
+    def self.licenses(project)
+      return project['info']['license'] unless project['info']['license'] == ''
```

Comment:
You can use `.present?` here instead of comparing to empty string, will also catch nulls then